phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/tenantoperation/PreScenarioOperationSupplier.java

```diff
@@ -48,32 +49,34 @@ public PreScenarioOperationSupplier(PhoenixUtil phoenixUtil, DataModel model, Sc
 
             @Override
             public OperationStats apply(final TenantOperationInfo input) {
+                Preconditions.checkNotNull(input);
                 final PreScenarioOperation operation = (PreScenarioOperation) input.getOperation();
-                final String tenantId = input.getTenantId();
                 final String tenantGroup = input.getTenantGroupId();
                 final String opGroup = input.getOperationGroupId();
                 final String tableName = input.getTableName();
                 final String scenarioName = input.getScenarioName();
-                final String opName = String.format("%s:%s:%s:%s:%s",
-                        scenarioName, tableName, opGroup, tenantGroup, tenantId);
 
                 long startTime = EnvironmentEdgeManager.currentTimeMillis();
                 int status = 0;
                 if (!operation.getPreScenarioDdls().isEmpty()) {
-                    try (Connection conn = phoenixUtil.getConnection(tenantId)) {
-                        for (Ddl ddl : operation.getPreScenarioDdls()) {
-                            LOGGER.info("\nExecuting DDL:" + ddl + " on tenantId:" + tenantId);
+                    for (Ddl ddl : operation.getPreScenarioDdls()) {
+                        final String tenantId = ddl.isUseGlobalConnection() ? null : input.getTenantId();
+                        final String opName = String.format("%s:%s:%s:%s:%s",
+                                scenarioName, tableName, opGroup, tenantGroup, input.getTenantId());
+
+                        try (Connection conn = phoenixUtil.getConnection(tenantId)) {
+                            LOGGER.info("\nExecuting DDL:" + ddl + ", OPERATION:" + opName);
                             phoenixUtil.executeStatement(ddl.toString(), conn);
                             if (ddl.getStatement().toUpperCase().contains(phoenixUtil.ASYNC_KEYWORD)) {
                                 phoenixUtil.waitForAsyncIndexToFinish(ddl.getTableName());
                             }
+                        } catch (SQLException sqle) {
```

Comment:
Looks like you don't need this catch block since the one below does the same thing and covers it

Reply:
@jpisaac ping